src/stdlib/parse_etld.rs

```diff
@@ -137,6 +137,8 @@ impl FunctionExpression for ParseEtldFn {
 
         let plus_parts = match self.plus_parts.resolve(ctx)?.try_integer()? {
             x if x < 0 => 0,
+            #[allow(clippy::cast_sign_loss, clippy::cast_possible_truncation)]
+            // TODO consider removal options
```

Comment:
Nit I'd find this slightly easier to read if the TODO was above the clippy allow.